src/SelectPicker/test/SelectPickerSpec.js

```diff
@@ -400,4 +400,24 @@ describe('SelectPicker', () => {
       expect(getByTestId('content')).to.have.text('Not selected');
     });
   });
+  it('SearchWord should be reset when controlled and triggered off', async () => {
+    const handleSearch = sinon.spy();
+    const Wrapper = () => {
+      return (
+        <>
+          <button id="exit">exit</button>
+          <SelectPicker container={container} defaultOpen onSearch={handleSearch} data={data} />
+        </>
+      );
+    };
+    Wrapper.displayName = 'WrapperSelectPicker';
+    let { container } = render(<Wrapper />);
```

Comment:
Unnecessary wrapping. Simply `render` the fragment.

Reply:
Yeah